dotCMS/src/integration-test/java/com/dotcms/graphql/business/GraphqlAPITest.java

```diff
@@ -522,7 +525,8 @@ public void testGetSchema_FieldOperations(final TypeTestCase testCase) throws Do
         final GraphQLFieldDefinition fieldDefinition =
                 schema.getObjectType(testCase.contentTypeName).getFieldDefinition(testCase.fieldVarName);
 
-        GraphQLOutputType expectedType = api.getGraphqlTypeForFieldClass(
+        GraphQLOutputType expectedType = ContentAPIGraphQLTypesProvider.INSTANCE
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'expectedType' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5642169)